02-oop-inheritance/javaScript/refactored_JS/downloableMovie v0.1.js

```diff
@@ -0,0 +1,94 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function ( title ) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function( title ) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+var Movie = function ( paramName, paramValue ) {
+  //Variables
+  this.attributes = [];
+  this.attributes[paramName] = paramValue; 
+  this.observer = new MovieObserver();
+};//end movie constructor
+
+Movie.prototype = {
+  
+  get: function (atribute) {
+     return this.attributes[atribute];
+  }, 
+
+  set:function (atribute, value) {
+    this.attributes[atribute] = value;
+      
+  },
+
+  play:function () {
+    console.log('This is Movie.play method: I am a movie. This is my title: ' + this.attributes['title']);
+    this.observer.playing(this.attributes['title']);
```

Comment:
I've tried and it works but doesn't seems right. The attributes should be an Object instead.